calico_node/tests/st/bgp/test_ipip.py

```diff
@@ -57,8 +58,8 @@ def test_ipip(self, backend):
             # v1.0.2 calicoctl.  For calicoctl v1.1.0+, a new IPIP mode field
             # is introduced - by testing with an older pool version validates
             # the IPAM BIRD templates function correctly without the mode field.
-            self.pool_action(host1, "create", DEFAULT_IPV4_POOL_CIDR, False,
-                           calicoctl_version="v1.0.2")
+            self.pool_action(host1, "create", DEFAULT_IPV4_POOL_CIDR, ipip_mode="Never",
```

Comment:
So according to the test comment it seems like we're testing the older IPIPMode values, so if we're using older calicoctl here, it won't understand the mode `Never`, it should however understand mode `off` (lowercase)

Reply:
The comment is wrong.  This whole suite should only be testing the V2 API

Reply:
I'm not sure what our original intent was for adding this, but I'm guessing this could be useful when we have the migration scripts from old to new versions